packit/local_project.py

```diff
@@ -135,15 +154,6 @@ def __init__(
                 self.namespace = self.git_project.namespace
                 change = True
 
-            if self.git_repo and not self.git_url:
-                # this is prone to errors
-                # also if we want url to upstream, we may want to ask for it explicitly
-                # since this can point to a fork
-                # .urls returns generator
-                self.git_url = list(self.git_repo.remote().urls)[0]
-                logger.debug("remote url of the repo is %s", self.git_url)
-                change = True
```

Comment:
This move is not necessary -- when you set `change = True`, you will go to the next iteration of `while` and check the conditions once again.